app/readme.md

```diff
@@ -11,9 +11,6 @@ To **run the app locally**, you need to do **four things**:
 1. Get the code, navigate to the `app` folder and install `docker` and `docker-compose`
 2. Compile the protocol buffers
 3. Launch the backend with `docker-compose`
-4. Install and launch the web frontend with `yarn`
-
-Are you only developing on the web frontend? If you don't want to install docker, you can follow the alterative instructions in `web/readme.md`.
```

Comment:
I would keep this, but update it to say something like:
Are you only developing on the web frontend? You can run the frontend by itself, just follow instructions outlined in "add link here"